src/templates/_components/widgets/AcclaroAds/body.twig

```diff
@@ -0,0 +1,28 @@
+{% import 'translations/_components/_forms/button' as button %}
+
+<div class="recentnews-container scroll-x-auto">
+    {% if ads|length %}
+        <div class="data ad-container fullwidth">
+            {% for ad in ads %}
+                <div class="field">
+                    <div class="heading">
+                        <label> {{ ad.heading }} </label>
+                    </div>
+                    <div class="input ltr">
+                        {{ ad.content }}
+                    </div>
+                    <div class="flex" style="justify-content: end;">
+                        {{ button.anchorButton({
+                            label: ad.linkText,
+                            href: ad.link,
+                        }) }}
+                    </div>
+                </div>
+            {% endfor %}
+        </div>
+    {% else %}
+        <div class="data fullwidth">
+            <p>{{ "No ads to show."|t('app') }}</p>
```

Comment:
If there aren't any available, we should just not display anything.

Reply:
done